### PR TITLE
Moved "intrument" to under "http"

### PR DESCRIPTION
**Pull request by Nyholm**
https://github.com/ekino/EkinoNewRelicBundle/blob/master/DependencyInjection/Configuration.php#L96

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -93,6 +92,7 @@ ekino_new_relic:
         using_symfony_cache: false        # Symfony HTTP cache (see below) (default: false)
         transaction_naming: route         # route, controller or service (see below)
         transaction_naming_service: ~     # Transaction naming service (see below)
+        instrument: false                     # If true, uses enhanced New Relic RUM instrumentation (see below) (default: false)
```
Comment (jderusse) [inline README.md]: Bad indentation of comments

**Comment by jderusse** [comment]
Is it only related to http? I mean it does not make sense for commands ?

**Comment by Nyholm** [comment]
It is related to Twig as far as I know. So.. both HTTP and possible Commands... Or none. 

This PR just make sure the readme is up to date with the code.

**Comment by Nyholm** [comment]
Thank you. I wrote an issue

**Pull request merged**